lib/src/command/login.dart

```diff
@@ -60,5 +60,5 @@ class _UserInfo {
   final String email;
   _UserInfo(this.name, this.email);
   @override
-  String toString() => '<$email> "$name"';
+  String toString() => '<$email> ${name ?? ''}';
```

Comment:
I don't like the extra space we get when name is null. Perhaps:

```suggestion
  String toString() => ['<$email>', if (name != null) name].join(' ');
```